--- monosim/images/monosim.desktop.orig	2010-05-01 17:21:33.000000000 +0200
+++ monosim/images/monosim.desktop	2010-05-01 17:22:06.000000000 +0200
@@ -1,16 +1,13 @@
 [Desktop Entry]
 Categories=Office;ContactManagement;GTK;GNOME;
-Comment[it]=monoSIM - gestisce i contatti sulla tua SIM Card
 Comment=monoSIM - manage your SIM Card contacts
+Comment[it]=monoSIM - gestisce i contatti sulla tua SIM Card
 Comment[it_IT]=monoSIM - gestisce i contatti sulla tua SIM Card
 Exec=monosim
-GenericName[it]=monosim
 GenericName=monosim
 Icon=monosim
 MimeType=
-Name[it]=monosim
 Name=monosim
-Name[it_IT]=monosim
 StartupNotify=true
 Terminal=false
 Type=Application
